core/resourceid/id.go

```diff
@@ -0,0 +1,33 @@
+package resourceid
```

Comment:
This got split out to its own package as part of clipping dependencies from the CLI to various buildkit/containerd packages